### DISCUSSION TITLE
Show gap to car behind for leader on the OBC camera timing info

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#185] [Feature requests]
Currently due to a quirk in the timing, if you are watching a driver who is in the lead it will not show the gap to 2nd place as the timing gap is intended for cars ahead.

Maybe some sort of override is needed to show the gap from the car in P2 when a car is in P1.

**Comment by Turrican76** [comment]
I would like to add that watching a midfield driver i don`t really need the gap to leader. It would be more interesting to see gap to car in front and behind. So maybe this could be solved with a toggle (keyboard shortcut) which switches what is shown between "Gap to leader" and "Gap behind".

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #283
